src/effects/effectchainslot.cpp

```diff
@@ -139,18 +422,9 @@ void EffectChainSlot::slotChainNameChanged(const QString&) {
     emit(updated());
 }
 
-void EffectChainSlot::slotChainEnabledChanged(bool bEnabled) {
-    m_pControlChainEnabled->set(bEnabled);
-    emit(updated());
-}
-
-void EffectChainSlot::slotChainMixChanged(double mix) {
-    m_pControlChainMix->set(mix);
-    emit(updated());
-}
-
-void EffectChainSlot::slotChainMixModeChanged(EffectChainMixMode mixMode) {
-    m_pControlChainMixMode->set(static_cast<double>(mixMode));
+void EffectChainSlot::slotChainUpdated(double v) {
```

Comment:
I think this function is unnecessary. Can't you just connect the signals to `sendParameterUpdate`?